### PR TITLE
Rules: Create new string validation rule

### PR DESCRIPTION
**Pull request by markdunphy**
Add a simple validation rule to determine if a value is a string or not.

Reviewers:
- @bryanlatten 
- @ingluisjimenez

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,14 +90,15 @@ else {
 * alpha
 * alphaNumeric
 * array
-* email
-* phone
 * decimal
+* email
```
Comment (markdunphy) [inline README.md]: Just alphabetized and added `string`

**Comment by bryanlatten** [comment]
:+1:

**Comment by ingluisjimenez** [comment]
:+1:

File: tests/Validation/Rules/StringRuleTest.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @group validation
+ */
+class NBD_Validation_Rules_StringRuleTest extends PHPUnit_Framework_TestCase {
+
+  protected $_class = '\Behance\NBD\Validation\Rules\StringRule';
```
Comment (markdunphy) [inline tests/Validation/Rules/StringRuleTest.php]: No `::class` since we support PHP 5.4

**Pull request merged**